pdns/tcpiohandler.cc

```diff
@@ -951,6 +955,7 @@ class OpenSSLTLSIOCtx: public TLSCtx, public std::enable_shared_from_this<OpenSS
   std::shared_ptr<SSL_CTX> d_tlsCtx{nullptr}; // client context, on a server-side the context is stored in d_feContext->d_tlsCtx
   std::unique_ptr<OpenSSLFrontendContext> d_feContext{nullptr};
   bool d_ktls{false};
+  pdns::UniqueFilePtr d_keyLogFile{nullptr};
```

Comment:
Just a nit, please move the pointer above the boolean so we don't waste space due to padding:
```suggestion
  pdns::UniqueFilePtr d_keyLogFile{nullptr};
  bool d_ktls{false};
```